evalml/tests/utils_tests/test_dependencies.py

```diff
@@ -1,17 +1,24 @@
 import pathlib
 from importlib import import_module
 
+import pytest
 import requirements
 
 
-def test_has_minimal_deps(has_minimal_dependencies):
+def test_has_minimal_deps(has_minimal_dependencies, is_running_py_39_or_above):
     reqs_path = pathlib.Path(__file__).absolute().parents[3].joinpath('requirements.txt')
     lines = open(reqs_path, 'r').readlines()
     lines = [line for line in lines if '-r ' not in line]
     reqs = requirements.parse(''.join(lines))
     extra_deps = [req.name if req.name != 'imbalanced-learn' else 'imblearn' for req in reqs]
     extra_deps += ['plotly.graph_objects']
     for module in extra_deps:
+        # We don't expect to install sktime in python 3.9. Let's verify it's not present:
+        if module == 'sktime' and is_running_py_39_or_above:
+            with pytest.raises(ModuleNotFoundError):
+                import_module(module)
+            continue
```

Comment:
Got it. So the reason this doesn't appear in our coverage report is because we're currently only generating coverage on python 3.8. 👍